api/hasura/actions/_handlers/createUserMutation.ts

```diff
@@ -0,0 +1,99 @@
+import { ValueTypes } from '../../../../api-lib/gql/__generated__/zeus';
+import { adminClient } from '../../../../api-lib/gql/adminClient';
+import { UnprocessableError } from '../../../../api-lib/HttpError';
+
+async function checkExistingUser(address: string, circleId: number) {
+  const { users: existingUsers } = await adminClient.query(
+    {
+      users: [
+        {
+          limit: 1,
+          where: {
+            address: { _ilike: address },
+            circle_id: { _eq: circleId },
+          },
+        },
+        {
+          id: true,
+          deleted_at: true,
+        },
+      ],
+    },
+    {
```

Comment:
can we avoid splitting objects into multiple lines where they would fit on one?

Reply:
Sorry i just follow whatever the linter does, maybe we can change it?

Reply:
@levity actually i can't really figure out what you mean here, maybe the source changed?